File: 0001 Code Chef/0021 Biryani classes/implementation/0006 php/biryani_classes.php
<?php
    $test = (int) fgets(STDIN);
    while($test > 0){
        $stringInput = fgets(STDIN);
        $stringArray = explode(" ",$stringInput);
        $x = (int) $stringArray[0];
        $y = (int) $stringArray[1];
        $total = $x * $y;
        echo "$total\n";
        $test = $test - 1;
    }
?>

File: 0001 Code Chef/0021 Biryani classes/implementation/0001 c/biryani_classes.c
#include <stdio.h>

int main(){
    int test;
    scanf("%d",&test);
    while(test > 0){
        int x,y,total;
        scanf("%d %d",&x,&y);
        total = x * y;
        printf("%d\n",total); 
        test = test -1;
    }
    return 0;
}